docs/competition/team-support/battery-charging.md

```diff
@@ -7,44 +7,76 @@ original:
 
 We (Student Robotics) manage the charging of all batteries at the competition. Battery Charging volunteers are responsible for ensuring that there is a continuous supply of charged batteries available for use by teams.
 
-## Equipment
+## Setup
 
- * Battery chargers
- * Discharged battery box
- * Charged battery box
- * Damaged battery/charger box
- * Red insulating tape
- * Example good batteries (one brand new, one squidgy)
+### Equipment
+
+There are two locations involved in battery charging:
+
+- Helpdesk
+    - 1 x Discharged Battery Box (18L RUB), clearly labelled
+    - 1 x Charged Battery Box (18L RUB), clearly labelled
+    - 1 x LiPo Bag for damaged batteries, clearly labelled, red taped
+    - At least one roll of red insulating tape
+- Battery Charging Station
+    - At least `n` battery chargers
+    - Enough LiPo charging bags for every charger.
+    - 1 x Discharged Battery Box (18L RUB), clearly labelled
+    - 1 x Charged Battery Box (18L RUB), clearly labelled
+    - 1 x Damaged battery box (18L RUB), clearly labelled, red taped
+    - At least one roll of red insulating tape
+
+
+### Safe Use of Battery Charging Equipment - Battery Charging Station 
+
+- There should be `0.5A` of mains electricity available per charger.
+- It should be ensured that any power distribution / 4 ways is setup properly and able to handle the current draw.
+- Additionally, there should be no trailing wires across the floor to prevent charging batteries falling on the floor.
 
 ## Volunteer Requirements
 
-Every volunteer performing the Battery Charging role must have read this document fully. Any doubts must be discussed with Rich Barlow ASAP.
+Every volunteer performing the Battery Charging role must have read this document fully. Any doubts must be raised before beginning the role.
```

Comment:
This implies that doubts which emerge after beginning are prohibited, which could lead to people ignoring a problem. Please could we keep the earlier wording (even if we change the person to contact).

Reply:
I plan to change this to a role title, rather than having a name. Is the team support co-ordinator still a role, or should this be replaced with a different role name?

Reply:
I'm not sure, though I'm not sure that's necessarily the right person anyway. It needs to be whoever is doing the training for this role, though I don't think we have a specific name for that.

Reply:
Should I possibly just direct this to health and safety aswell?

Reply:
My interpretation of this paragraph is that it's for things which the volunteer is unsure about in regards to how they should perform the role. I'm not sure that the Health and Safety officer is a good person for them to ask questions of here -- the person who trained them is a much better person to ask.